tiatoolbox/tools/pyramid.py

```diff
@@ -144,6 +143,7 @@ def get_tile(
         level: int,
         x: int,
         y: int,
+        res: int = 1,
         pad_mode: str = "constant",
         interpolation: str = "optimise",
```

Comment:
If we are standardizing on US spelling shall we use "optimize"?
```suggestion
        interpolation: str = "optimize",
```

Reply:
It seems the most common usage is 'optimise'; for example optimise is used everywhere in wsireader.py, and pyramid.py. So maybe it would be best to leave it as is for the moment.